drydock/templates/drydock/k8s/lifecycle/lms.yml

```diff
@@ -7,21 +7,41 @@ spec:
     spec:
       containers:
         - name: lms
-          lifecycle:
```

Comment:
Do you have any reason to remove the lifecycle hook? I think we can reduce this hook to a wait of 15 seconds and set `terminationGracePeriodSeconds` to 30 (the default) to reduce the termination waiting time.

Reply:
Based on our testing it seems this is not necessary for UWSGI deployments as the hook applied will first terminate gracefully the workers and then kill himself.

However, the should keep the `terminationGracePeriodSeconds`.